### ISSUE TITLE
setup.py incompatible with pip 10

### ISSUE THREAD

**Issue by OddBloke** [fginther/review-gator#18]
```
$ pip install --upgrade pip
Requirement already up-to-date: pip in /home/daniel/.virtualenvs/review-gator/lib/python3.6/site-packages (10.0.1)

$ pip install -e .
Obtaining file:///home/daniel/dev/review-gator
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/daniel/dev/review-gator/setup.py", line 6, in <module>
        from pip.req import parse_requirements
    ModuleNotFoundError: No module named 'pip.req'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /home/daniel/dev/review-gator/
```

**Comment by OddBloke** [comment]
A common workaround would be to just readlines the requirements file, but the fact we have editable dependencies means that the trivial implementation of this doesn't work.

**Comment by fginther** [comment]
This has been addressed in #20.
